### PR TITLE
Change footer social icon spacing and add text

### PR DESCRIPTION
**Pull request by AdamMankowski**
## TO DO:
* [X] Change spacing
* [X]  Add text next to a social icon

---

Review:
* [ ] @beatadelura
* [ ] @kruszczynski
* [x] @pjar
* [ ] @rpietraszko
* [x] @MagdaMalinowska
* [ ] @aberracja

### CODE REVIEW

File: source/stylesheets/modules/_m-footer.scss [hunk 1]
```diff
@@ -235,9 +234,63 @@
...
+      display: inline-block;
+      margin-left: 15px;
+      vertical-align: top;
+    }
+
+    .facebook-footer {
+      &:hover {
+        color: $breaker-bay-green;
+          svg {
+            fill: $breaker-bay-green;
```
Comment (MagdaMalinowska) [inline source/stylesheets/modules/_m-footer.scss]: Is it needed? svg should use currentColor by default

**Comment by AdamMankowski** [comment]
@MagdaMalinowska we have changed that in PP today. I'm going to merge that now and then resolve conflicts on new branch.

File: source/partials/_footer.html.slim [hunk 2]
```diff
@@ -35,19 +35,21 @@ footer.m-footer
...
+        a.github-footer(href='https://github.com/codequest-eu'
+                        target='_blank' rel='nofollow')
+          li.social-link-item
+            .social-link.github-link = partial 'svg/main_menu/github_icon'
+            .social-link.footer-social-text Check our code
+        a.twitter-footer(href='https://twitter.com/codequest'
+                      target='_blank' rel='nofollow')
+          li.social-link-item
+            .social-link.twitter-link = partial 'svg/main_menu/twitter_icon'
+            .social-link.footer-social-text Check what we read
```
Comment (aberracja) [inline source/partials/_footer.html.slim]: Why are texts different than in trello? Those from trello are much better.

**Comment by AdamMankowski** [comment]
good catch i forgot to change it 👍

**Pull request merged**